genie/process_functions.py

```diff
@@ -323,6 +323,81 @@ def center_convert_back(filePath, anonymizeCenterDf):
 ####################################################################################
 # UPDATING DATABASE
 ####################################################################################
+def _append_rows(new_datasetdf, databasedf, checkby):
+	#All new rows
+	databasedf.fillna('',inplace=True)
+	new_datasetdf.fillna('',inplace=True)
+	if not databasedf.empty:
+		appenddf = new_datasetdf[~new_datasetdf[checkby].isin(databasedf[checkby])]
+	else:
+		appenddf = new_datasetdf.copy()
+	if not appenddf.empty:
```

Comment:
confirm what the backend will do when you upload a csv without ROW_ID and ROW_VERSION. I think it would just add the new rows.

Reply:
No need for ROW_ID and ROW_VERSION